libs/sdk-backend-spi/src/workspace/ldm/catalog.ts

```diff
@@ -234,6 +240,15 @@ export interface IWorkspaceCatalogFactoryMethods<TFactory, TOptions> {
      */
     excludeTags(tags: ObjRef[]): TFactory;
 
+    /**
+     * Setup whether catalog should fetch groups.
+     * Default: true
+     *
+     * @param loadGroups - should fetch groups
+     * @returns catalog factory
+     */
+    withGroups(loadGroups: boolean): TFactory;
```

Comment:
We should make sure the `groups()` call prints some error, or even throws if the factory was set up with groups not loading. WDYT?

Reply:
It makes sense, but this is just a merge to hotfix branch, could we keep it for my PR to master?